src/smt/env.cpp

```diff
@@ -51,6 +51,7 @@ Env::Env(NodeManager* nm, const Options* opts)
   {
     d_options.copyValues(*opts);
   }
+  d_statisticsRegistry->registerTimer("driver::totalTime").start();
```

Comment:
Maybe we shouldn't call it `driver::totalTime` anymore since the timer does not live in the driver anymore? How about `global::totalTime`?

Reply:
Done.